db/migrate/20180321000000_change_sync_status_to_sync_task.rb

```diff
@@ -0,0 +1,7 @@
+class ChangeSyncStatusToSyncTask < ActiveRecord::Migration
+  def change
+    remove_column :scc_accounts, :sync_status, :string
+    add_column :scc_accounts, :sync_task_id, :uuid, null: true
+    # add_foreign_key "scc_accounts", "foreman_tasks_tasks", name: "scc_accounts_foreman_tasks_tasks_id_fk", column: "sync_task_id", primary_key: "id" , on_delete: :nullify
```

Comment:
If i am getting this right, we are unable to set this PK-constraint, because the id field of ForemanTasks::Task (type uuid) is not flagged unique. :disappointed: